src/puppetlabs/trapperkeeper/authorization/ring.clj

```diff
@@ -23,3 +15,36 @@
    schema/Keyword schema/Any})
 
 ;; Functions
+
+(schema/defn get-authorized-authentic? :- schema/Bool
```

Comment:
Yeah, it lines up with the rest of Ring's functions which extract things from the request.